react/features/welcome/components/WelcomePage.native.js

```diff
@@ -261,7 +262,9 @@ class WelcomePage extends AbstractWelcomePage {
                     <Header style = { styles.header }>
                         <TouchableOpacity onPress = { this._onShowSideBar } >
                             <Icon
-                                name = 'menu'
+                                color = { _headerStyles.iconColor }
+                                size = { HEADER_ICON_SIZE }
+                                src = { IconMenu }
```

Comment:
I wonder if the api would be better if IconMenu could be used by itself somehow. So it would be <IconMenu {...props} /> instead of <Icon {...props} />. Would that be too much boilerplate or maybe there could be code to automatically generate the components, but then it might be non-obvious looking code. Right now Icon and the various SVGs are closely related.

Reply:
I'm wondering the same for every icon. IMHO the Icon class is not bringing too much value for now. It basically adds a Container to the svg. I'm wondering if we really need the Icon class. Maybe it will be simpler to use the svgs directly. WDYT?

Reply:
The loaded SVGs are individual components on both platforms, so they can be used as a simple component if need be, not wrapped into an ```Icon``` component.

However the ```Icon``` component provides cross-platform compatibility and avoids us to repeat props, such as ```width``` and ```height``` combined into a single ```size``` prop with a default value (the standalone SVG component uses the hardcoded size info from the SVG file without this), and a default color (which would also be the default hardcoded value from the SVG file otherwise). Also on mobile we need the additional View to format the icon in some cases.

Reply:
<IconMenu /> could have an <Icon />, so the question for me becomes if it's worthwhile to consume the icons through one component instead of two.